sky/packages/sky/lib/gestures/arena.dart

```diff
@@ -32,6 +32,8 @@ class GestureArenaEntry {
 class GestureArena {
   final Map<Object, List<GestureArenaMember>> _arenas = new Map<Object, List<GestureArenaMember>>();
 
+  static final GestureArena instance = new GestureArena();
```

Comment:
It seems sad for it to own its own unique instance. Shouldn't the binding own it or something?